internal/metrics/metrics.go

```diff
@@ -82,11 +84,35 @@ func NewGinMiddleware(metrics *Metrics) gin.HandlerFunc {
 			).Observe(duration)
 
 			metrics.responseSizes.WithLabelValues(path, status).Observe(size)
-			metrics.requestCount.WithLabelValues(method, path).Inc()
+			metrics.updateRequestCountMetric(method, path, vdsURLs)
 		}()
 	}
 }
 
+func extractStorageAccounts(vdsURLs []string) []string{
+	var storageAccounts []string
+	for _, vdsURL := range vdsURLs {
+		parsedURL, err := url.Parse(vdsURL)
+		if err != nil {
+			storageAccounts = append(storageAccounts, vdsURL)
+			continue
+		}
+		storageAccounts = append(storageAccounts, parsedURL.Hostname())
+	}
+
+	slices.Sort(storageAccounts)
```

Comment:
Why are these sorted? Is the user expect these to be returned in alphabetical order?

Reply:
Because apparently by sorting and using compact one can remove duplicates using slice package.

Reply:
Ok, fine.